packages/apps/kadena-docs/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kadena/kadena-docs",
+  "version": "0.1.0",
+  "scripts": {
+    "_phase:build": "next build",
+    "_phase:test": "",
```

Comment:
Maybe use `npm run build` and `npm run test` that way we have the same build local as in CI